query-planner-wasm/Cargo.toml

```diff
@@ -1,7 +1,6 @@
 [package]
 name = "query-planner-wasm" # this ends up being the name in the published npm package.
 version = "0.0.0-DO.NOT.CHANGE" # The version is managed in the package.json, by Lerna.
-private = true
```

Comment:
Is this intentional? I suspect @abernix could speak to its existence. This seems likely to be a relic of `package.json`'s `private` key - maybe it's reflected in a generated `package.json`?

Reply:
afaict this had no affect, and in fact caused a warning that said there's an unknown key in the file.
Also, for future reference, the only keys that get copied to the `package.json` that gets generated by `wasm-pack` are [here](https://github.com/rustwasm/wasm-pack/blob/master/src/manifest/npm/commonjs.rs)

Reply:
Yeah, all good to remove this; this doesn't appear to do what I had thought it would.  I was trying to prevent this package from getting accidentally published to Crates.io.  Though I think that what I had meant to do was probably to use the [`publish = false`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field) option.